AdMobBuddy.Android/AdCallback.cs

```diff
@@ -31,9 +34,8 @@ public override void OnAdDismissedFullScreenContent()
 			Debug.WriteLine("Ad closed.");
 
 			base.OnAdDismissedFullScreenContent();
-
-			Adapter.LoadInterstitialAd();
```

Comment:
After an interstitial ad is dismissed, this line of code queues up the next one. Any reason it was removed?

Reply:
Yep. The load has been moved to the event handler.